api/src/main/java/org/pmiops/workbench/api/util/SQLGenerator.java

```diff
@@ -28,15 +28,15 @@
                     "is_selectable,\n" +
                     "domain_id\n" +
                     "from `%s.%s.%s`\n" +
-                    "WHERE parent_id = @parentId\n" +
+                    "where parent_id = @parentId\n" +
```

Comment:
WHERE can stay all caps.

Reply:
I change the entire sql strings to be lowercase.

Reply:
data is fixed